app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,58 @@
+     <p><a href="/life-as-a-teacher/explore-subjects">Explore some subjects you could teach</a>.</p>
+     <%= render 'content/shared/teaching/ages_abilities' %>
+     <h2 class="heading--box-blue">What you need to teach in secondary schools</h2>
+     <p>You need a bachelor’s degree in any subject to train to teach secondary in England. If you do not already have a degree, you can train to be a teacher as part of your bachelor’s degree and get QTS.</p>
+     <p>If you already have a degree or are training for one, you can <a href="/train-to-be-a-teacher/if-you-have-a-degree">find out what routes are available to you</a>.</p>
+     <p>If you are interested in teaching young people and adults over 16 who are not studying for a degree, then <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teaching in further education</a> might be for you.</p>
+     <h2 class="heading--box-blue">Start your journey into teaching <%= @front_matter["subject"] %></h2>
+    <!-- Check your qualifications-->
+    <%= render 'content/shared/age-groups-and-specialisms/check_qualifcations' %>
+    <%= render 'content/shared/qualifications-training/find_out_more' %>
+    <!-- Non-UK qualifications expander -->
+    <%= render 'content/shared/qualifications-training/non_uk_qualifications' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p>Depending on the subject you choose to teach, you could be eligible for a bursary or scholarship. <a href="/funding-and-support/scholarships-and-bursaries">Check what funding is available for your subject</a>.</p>
+    <%= render 'content/shared/teaching/funding_support' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+   <%= render 'content/shared/qualifications-training/get_school_experience' %>
+   <div class="paid-internships">
+   <h3>Apply for a paid internship</h3>
+   <p>If you're currently doing an undergraduate or master's degree and are interested in chemistry, computing, langauges, maths or physics you could apply for a <a href="/train-to-be-a-teacher/teaching-internships">paid teaching internship</a>.</p>
+   <p>The 3-week programme could help you to understand what it's really like in the classroom and get a feel for school life.</p></div>
+  </section>
+  <section class="col col-720">
+    <%= render 'content/shared/teaching/ect_support' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Progression in teaching</h2>
+    <p>Once you have qualified to teach, there is still plenty of opportunity for <a href="/life-as-a-teacher/teaching-as-a-career/career-progression">progression in teaching</a>. You could take on challenges such as becoming a  head of year, head of subject, special educational needs co-ordinator or even a headteacher.</p>
```

Comment:
This might sit better under the what you'll teach etc bit. Could you make it a partial too maybe?